missing directory/packages for melodic

deb files forget to install `gundam_rx78_description/config`
we also find some missing/outdated packages on Melodic
```
[ WARN] [1581077969.862873700]: The 'state_publisher' executable is deprecated. Please use 'robot_state_publisher' instead
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-k-okada'
[WARN] [1581077970.493106]: The 'use_gui' parameter was specified, which is deprecated.  We'll attempt to find and run the GUI, but if this fails you should install the 'joint_state_publisher_gui' package instead and run that.  This backwards compatibility option will be removed in Noetic.
[ERROR] [1581077970.496683]: Could not find the GUI, install the 'joint_state_publisher_gui' package
[joint_state_publisher-1] process has died [pid 2174, exit code 1, cmd /opt/ros/melodic/lib/joint_state_publisher/joint_state_publisher __name:=joint_state_publisher __log:=/home/k-okada/.ros/log/756ed032-49a2-11ea-b863-f01dbc929497/joint_state_publisher-1.log].
log file: /home/k-okada/.ros/log/756ed032-49a2-11ea-b863-f01dbc929497/joint_state_publisher-1*.log
^C[rviz-3] killing on exit
```

--- a/gundam_rx78_description/launch/display.launch
+++ b/gundam_rx78_description/launch/display.launch
@@ -4,9 +4,8 @@
 
   <include file="$(find gundam_rx78_description)/launch/upload.launch" />
 
-  <param name="use_gui" value="$(arg gui)"/>
-
-  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" if="$(arg gui)" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" unless="$(arg gui)" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
   <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
 </launch>

--- a/gundam_rx78_description/package.xml
+++ b/gundam_rx78_description/package.xml
@@ -18,6 +18,7 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <exec_depend>joint_state_publisher</exec_depend>
+  <exec_depend>joint_state_publisher_gui</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>rviz</exec_depend>
 

